pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
This is surprising to me

Reply:
I'm not sure we want this enabled. This setting may have been set by pnpm based on the `.npmrc`. I swear I saw something like this come up in the past couple weeks.